src/utils/transferSend.ts

```diff
@@ -14,15 +14,12 @@ export function getMessageByFieldName(errorField?: string): IInputMessage {
   if (!errorField) {
     return { status: 'success' };
   }
-  if (errorField && WARNING_RULES_WORDING.includes(
+  if (WARNING_RULES_WORDING.includes(
     errorField.replace(noneAlphabeticCharsRegExp, '')
```

Comment:
Maybe by the way you can update the typo here? There is no such a thing as `noneAlphabetic...`. It should be `nonAlphabetic...` (`none` => `non`).